testing/regress/ecl/exampleplugin.ecl

```diff
@@ -0,0 +1,28 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2015 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+//class=embedded
```

Comment:
Theoretically the 3rdparty means there should be an installed 3rdparty server (mysql, etc.) or similar stuff to execute that ECL.
The embedded means any plugin like stuff.

These tags are used by the regression test engine to include/exclude ECL test case and managed manually.